src/bpm/integration2/run_test.go

```diff
@@ -30,10 +31,20 @@ import (
 	"bpm/integration2/bpmsandbox"
 )
 
+const charset = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+
 var runcExe = flag.String("runcExe", "/var/vcap/packages/bpm/bin/runc", "path to the runc executable")
 var tiniExe = flag.String("tiniExe", "/var/vcap/packages/bpm/bin/tini", "path to the tini executable")
 var bpmExe = flag.String("bpmExe", "", "path to bpm executable")
 
+func randomString(length int) string {
```

Comment:
Would it make sense to give this func a name that's more indicative of it's purpose? Something like `uniqueJobName(prefix String) string` to convey _why_ this is needed?

Reply:
I added `generateJobName` below. I'm happy to change it to `uniqueJobName` if it's not very descriptive.

Reply:
I think `uniqueJobName`, or a comment explaining why would be helpful for our future selves.

I know I'll forget why we do this between now and the next time I look at the code.

Probably also fine to inline `randomString` into the other method, unless we need it separately?